src/main/java/games/strategy/triplea/settings/ClientSettingSwingUiBinding.java

```diff
@@ -170,7 +171,14 @@
       SettingType.NETWORK_PROXY,
       SelectionComponentFactory.proxySettings(),
       "Configure TripleA's Network and Proxy Settings\n"
-          + "This only effects Play-By-Forum games, dice servers, and map downloads.");
+          + "This only effects Play-By-Forum games, dice servers, and map downloads."),
+
+  USE_EXPERIMENTAL_JAVAFX_UI(
+      "Use JavaFX UI (Incomplete!)",
```

Comment:
I'm not sure if the "(Incomplete!)" qualifier is necessary in the label.  The description makes it clear this feature is Use At Your Own Risk. :smile:

Reply:
Just wanted to make it extra clear, because nobody reads the description right 😄